htword/WordKeyInfo.cc

```diff
@@ -45,7 +45,7 @@ int WordKeyField::SetNum(WordKeyField *previous, char *nname, int nbits)
 
 int WordKeyField::SetString()
 {
-  name.set("Word");
+  strcpy(name, "Word");
```

Comment:
Please see http://www.tutorialspoint.com/cprogramming/c_structures.htm about variables with periods in them.

Reply:
Sorry! I didn't realize it was a struct!